### PR TITLE
Fixes #34696 - rename ssh to script provider

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by jlsherrill** [comment]
this will likely fail until the next REX release

File: db/seeds.d/150-module_job_templates.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 if Katello.with_remote_execution?
   User.as_anonymous_admin do
     JobTemplate.without_auditing do
-      module_template = JobTemplate.find_by(name: 'Module Action - SSH Default')
+      module_template = JobTemplate.find_by(name: 'Module Action - Script Default')
```
Comment (chris1984) [inline db/seeds.d/150-module_job_templates.rb:4]: Do we need a migration to change the name as well for existing installs?

Comment (adamruzicka) [reply db/seeds.d/150-module_job_templates.rb:4]: It is in rex

**Comment by adamruzicka** [comment]
Rex 6.1.0 is now out

**Review by chris1984 — APPROVED** [review]

**Comment by chris1984** [comment]
ACK

**Pull request merged**